ui/app/templates/components/transit-form-show.hbs

```diff
@@ -63,7 +63,7 @@
         />
       {{/if}}
       {{#if (eq @mode "show")}}
-        {{#if (or @capabilities.canUpdate @capabilities.canDelete)}}
```

Comment:
I don't even see a `@capabilities` arg passed into this component (and above)!? How the heck was this working before. I only looked because I was going to suggest cleaning up the unused args...I'm baffled 🤯